### PR TITLE
Add profile to website

### PR DESCRIPTION
**Pull request by Romain-Thomas-Shef**
Please check  my English :)

### CODE REVIEW

**Review by davidwyld — APPROVED** [review]
Looks good to me! Are we the RSE 'team' or 'group'? I've always thought we were 'team' but seems to be used interchangeably.

File: _people/romain-thomas.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Romain holds a PhD in Astrophysics and Cosmology carried out at the Laboratoire D'Astrophysique de Marseille, France. He then went for a postdoc at the Universidad de Valparaiso, Chile and became a fellow at the European Southern Observatory a couple of years later. During his research time, Romain focused on the study of high redshift galaxies using large galaxy survey conducted mainly at the VLT. Romain published various module/software related to his work in astronomy ([SPARTAN](https://ui.adsabs.harvard.edu/abs/2021A%26C....3400427T/abstract),[dfitspy](https://astrom-tom.github.io/dfitspy/build/html/index.html), [SEDOBS](https://ui.adsabs.harvard.edu/abs/2020A%26C....3000354T/abstract), [Photon](https://astrom-tom.github.io/Photon/build/html/index.html)).
+
+* Email: romain.thomas (at) Sheffield.ac.uk
+* Github [new]: [@Romain-Thomas-Shef](https://github.com/Romain-Thomas-Shef)
+* Github [old]: [@astrom-tom](https://github.com/astrom-tom)
+* Publications: [ADS link] (https://ui.adsabs.harvard.edu/user/libraries/lLg5JLTWRU2p8z0vp01YUQ)
```
Comment (Robadob) [inline _people/romain-thomas.md]: This link goes to a login page, I guess it's not public.

Comment (ns-rse) [inline _people/romain-thomas.md]:
```suggestion
* Publications: [ADS link](https://ui.adsabs.harvard.edu/user/libraries/lLg5JLTWRU2p8z0vp01YUQ) 
```

File: _people/romain-thomas.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+---
+
+Romain started as new head of the RSE group in August 2023. Prior to coming to the University of Sheffield he was a staff Astronomer at the European Southern Observatory (Chile) where he led the development of the [SCUBA software framework](https://ui.adsabs.harvard.edu/abs/2020SPIE11449E..0CT/abstract), a tool dedicated to control the quality of more than 15 Very Large Telescope (VLT) instruments.
+
+Romain holds a PhD in Astrophysics and Cosmology carried out at the Laboratoire D'Astrophysique de Marseille, France. He then went for a postdoc at the Universidad de Valparaiso, Chile and became a fellow at the European Southern Observatory a couple of years later. During his research time, Romain focused on the study of high redshift galaxies using large galaxy survey conducted mainly at the VLT. Romain published various module/software related to his work in astronomy ([SPARTAN](https://ui.adsabs.harvard.edu/abs/2021A%26C....3400427T/abstract),[dfitspy](https://astrom-tom.github.io/dfitspy/build/html/index.html), [SEDOBS](https://ui.adsabs.harvard.edu/abs/2020A%26C....3000354T/abstract), [Photon](https://astrom-tom.github.io/Photon/build/html/index.html)).
```
Comment (Robadob) [inline _people/romain-thomas.md]:
```suggestion
Romain holds a PhD in Astrophysics and Cosmology carried out at the Laboratoire D'Astrophysique de Marseille, France. He then went for a postdoc at the Universidad de Valparaiso, Chile and became a fellow at the European Southern Observatory a couple of years later. During his research time, Romain focused on the study of high redshift galaxies using large galaxy survey conducted mainly at the VLT. Romain has published various modules/software related to his work in astronomy ([SPARTAN](https://ui.adsabs.harvard.edu/abs/2021A%26C....3400427T/abstract),[dfitspy](https://astrom-tom.github.io/dfitspy/build/html/index.html), [SEDOBS](https://ui.adsabs.harvard.edu/abs/2020A%26C....3000354T/abstract), [Photon](https://astrom-tom.github.io/Photon/build/html/index.html)).
```

Only grammar that reads poorly to me.

File: _people/romain-thomas.md (hunk 1 above)
Comment (Robadob) [reply _people/romain-thomas.md]: Might just be worth linking your ORCID if that has all your publications linked?

File: _people/romain-thomas.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+level: 0
+published: true
+
+othernames: Romain
+surname: Thomas
+role: Head of group
+
+---
+
+Romain started as new head of the RSE group in August 2023. Prior to coming to the University of Sheffield he was a staff Astronomer at the European Southern Observatory (Chile) where he led the development of the [SCUBA software framework](https://ui.adsabs.harvard.edu/abs/2020SPIE11449E..0CT/abstract), a tool dedicated to control the quality of more than 15 Very Large Telescope (VLT) instruments.
```
Comment (ns-rse) [inline _people/romain-thomas.md]: Perhaps pedantic but rewording as below might age better as there will be a point where you're no longer new.

```suggestion
Romain joined the RSE Team as Head in August 2023. Prior to coming to the University of Sheffield he was a staff Astronomer at the European Southern Observatory (Chile) where he led the development of the [SCUBA software framework](https://ui.adsabs.harvard.edu/abs/2020SPIE11449E..0CT/abstract), a tool dedicated to control the quality of more than 15 Very Large Telescope (VLT) instruments.
```

File: _people/romain-thomas.md (hunk 1 above)
Comment (Romain-Thomas-Shef) [reply _people/romain-thomas.md]: Right, thanks!

Comment (Robadob) [reply _people/romain-thomas.md]: (Offtopic, I discovered you can [manually] add software DOIs to your ORCID last week, they don't sync to MyPublications though)

**Review by ns-rse — CHANGES_REQUESTED** [review]
Reads ok, a minor suggestion for the opening sentence but not essential.

Space in one of the links that @Robadob has made suggestions around.

File: _people/romain-thomas.md (hunk 3 above)
Comment (Romain-Thomas-Shef) [reply _people/romain-thomas.md]: Thanks @ns-rse

**Review by ns-rse — APPROVED** [review]
:+1:

**Review by ns-rse — APPROVED** [review]

**Pull request merged**